packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -43,6 +43,7 @@ const LIST_ITEM_SELECTED_CLASS = 'dx-list-item-selected';
 const STATIC_DELETE_BUTTON_CLASS = 'dx-list-static-delete-button';
 const TOGGLE_DELETE_SWITCH_CLASS = 'dx-list-toggle-delete-switch';
 const SWITCHABLE_DELETE_BUTTON_CLASS = 'dx-list-switchable-delete-button';
+const LIST_FOCUS_STATE_CLASS = 'dx-state-focused';
```

Comment:
This class is not specific for list
```suggestion
const FOCUSED_STATE_CLASS = 'dx-state-focused';
```

Reply:
[d899ec3](https://github.com/DevExpress/DevExtreme/pull/27230/commits/d899ec3ccf4d2ebd395b987bb76d4110ee11d357)